recipe/meta.yaml

```
@@ -1,26 +1,26 @@
 {% set name = "eth-typing" %}
-{% set version = "3.2.0" %}
-
+{% set version = "5.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 177e2070da9bf557fe0fd46ee467a7be2d0b6476aa4dc18680603e7da1fc5690
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 83debf88c9df286db43bb7374974681ebcc9f048fac81be2548dbc549a3203c0
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```

```suggestion
  number: 0
  skip: True  # [py<38]
```
https://github.com/ethereum/eth-typing/blob/432fe9eeb945416d8704d69c321235e0b6bd248d/setup.py#L52